backend/hitas/views/reports.py

```diff
@@ -0,0 +1,38 @@
+import datetime
+from typing import Any
+
+from django.http import HttpResponse
+from rest_framework import serializers
+from rest_framework.request import Request
+from rest_framework.viewsets import ViewSet
+
+from hitas.services.apartment_sale import find_sales_on_interval_for_reporting
+from hitas.services.reports import build_sales_report_excel
+from hitas.types import HitasJSONRenderer
+from hitas.views.utils.excel import ExcelRenderer, get_excel_response
+
+
+class SalesReportSerializer(serializers.Serializer):
+    start_date = serializers.DateField()
+    end_date = serializers.DateField()
+
+    def validate(self, attrs: dict[str, Any]) -> dict[str, Any]:
+        if attrs["start_date"] > attrs["end_date"]:
+            raise serializers.ValidationError("Start date must be before end date")
```

Comment:
🤔

Reply:
We can get this merged for now, and fix it in some later PR to avoid rebasing all of them 😄

Reply:
Isn't this correct? We are giving a range to search sales from, so the start of the range needs to be before the end of the range (or the same, if you just want one day).

Reply:
Yes nvm, brain fart 😅